### PR TITLE
Update changelog and add new contributor

### PR DESCRIPTION
**Pull request by bautrukevich**

### CODE REVIEW

**Comment by bautrukevich** [comment]
[release uploadcare-php version](https://trello.com/c/e6H9hmAA/225-release-uploadcare-php-version)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,17 +1,47 @@
...
-
-## 2.1.2
+All notable changes to this project will be documented in this file.
+
+The format is based now on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [2.2.0] - 2018-05-14
+### Added
+- Rights to user set own User-Agent
```
Comment (dmitry-mukhin) [inline CHANGELOG.md]: Allow user to specify User Agent

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,17 +1,47 @@
...
+- Getter $api->getUserAgentHeader() for retrieving HTTP User-Agent header
+- Getter $api->getIntegrationData() for data-integration attribute to the widget
+- Setters $api->setFramework($name, $version), $api->setExtension($name, $version), $api->setUserAgentName($name) methods reported during API requests
+- Getters $api->getFramework(), $api->getExtension(), $api->getLibraryName(), $api->getUserAgentName()
+- @throws to PHPDoc
```
Comment (dmitry-mukhin) [inline CHANGELOG.md]: lines 10-14 should not be here(this is not git log).
If you want to add this, rephrase and add as single item

e.g. changed: user agent reporting for the lib and integrations that use it

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,17 +1,47 @@
...
+- data-integration attribute to the widget
+- New contributor to AUTHORS.txt
```
Comment (dmitry-mukhin) [inline CHANGELOG.md]: this also should not be here

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,17 +1,47 @@
...
+
+### Changed
+- User-Agent reported during API requests
```
Comment (dmitry-mukhin) [inline CHANGELOG.md]: it wasn't reported?

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,17 +1,47 @@
...
+- Replace library version var with const, update PHPDoc for setters
```
Comment (dmitry-mukhin) [inline CHANGELOG.md]: this is redundant info

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,17 +1,47 @@
...
+- Changelog format
```
Comment (dmitry-mukhin) [inline CHANGELOG.md]: this is most likely redundant info

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,17 +1,47 @@
...
+- README.md accordingly to the last changes
```
Comment (dmitry-mukhin) [inline CHANGELOG.md]: this is redundant info

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,17 +1,47 @@
...
+
+### Fixed
+- $api->getGroupList($options = array()) method
+- Some mistakes in description of methods
+- An issue, that response status is a success but still hit max attempts
```
Comment (dmitry-mukhin) [inline CHANGELOG.md]: Hitting max throttling attempts if request was successful

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,17 +1,47 @@
...
+
+### Removed
+- Unnecessary imports
```
Comment (dmitry-mukhin) [inline CHANGELOG.md]: this is redundant info

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,17 +1,47 @@
...
+- New contributor to AUTHORS.txt
+
+### Changed
+- User-Agent reported during API requests
+- Replace library version var with const, update PHPDoc for setters
+- Changelog format
+- README.md accordingly to the last changes
+
+### Fixed
+- $api->getGroupList($options = array()) method
```
Comment (dmitry-mukhin) [inline CHANGELOG.md]: use ticks to format code

File: CHANGELOG.md (hunk 1 above)
Comment (bautrukevich) [reply CHANGELOG.md]: Ok

File: CHANGELOG.md (hunk 2 above)
Comment (bautrukevich) [reply CHANGELOG.md]: Ok

File: CHANGELOG.md (hunk 3 above)
Comment (bautrukevich) [reply CHANGELOG.md]: Only data-integration attribute to the widget need — it was added to the widget

File: CHANGELOG.md (hunk 4 above)
Comment (bautrukevich) [reply CHANGELOG.md]: Now is reporting in new format by default

File: CHANGELOG.md (hunk 5 above)
Comment (bautrukevich) [reply CHANGELOG.md]: Ok

File: CHANGELOG.md (hunk 7 above)
Comment (bautrukevich) [reply CHANGELOG.md]: ok

File: CHANGELOG.md (hunk 6 above)
Comment (bautrukevich) [reply CHANGELOG.md]: ok

File: CHANGELOG.md (hunk 8 above)
Comment (bautrukevich) [reply CHANGELOG.md]: Ok

File: CHANGELOG.md (hunk 9 above)
Comment (bautrukevich) [reply CHANGELOG.md]: Ok

**Comment by bautrukevich** [comment]
@dmitry-mukhin thank you!

**Pull request merged**